package-lock.json

```diff
@@ -1,8 +1,245 @@
 {
   "name": "@boardroom/protocol-info",
   "version": "2.10.18",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
```

Comment:
Why do we have these changes for readme file change?, can you remove them and just add readme commit?

Reply:
I do not know why , all I did was change the .md file

Reply:
Can you fix it?